Week1

mseiser opened the pull request:
All changes of first week

MountAnFruh committed:
Added prototype level with collsion detection

MountAnFruh committed:
Added doors to room

MountAnFruh committed:
Added Basic Level

KonstantinWindisch committed:
added player and simple shooting

KonstantinWindisch committed:
Merge branch 'feat_levels' into player_core_mechanics

KonstantinWindisch committed:
Made camera follow player and added collision with wall

Cezzel committed:
MainMenu

Main menu added including Sound, Grafics and Buttons.
Not included = Options Menu

mseiser committed:
Merge pull request #2 from DecentCat/player_core_mechanics

Player core mechanics

mseiser committed:
Delete unity/UserSettings directory

- UserSettings lead to unecassery merge conflicts
- Fodlers like UserSettings and Library should not be included in version control
-https://docs.unity3d.com/2020.1/Documentation/Manual/ExternalVersionControlSystemSupport.html

mseiser committed:
Delete unity/UserSettings directory

mseiser committed:
Update .gitignore

mseiser committed:
Merge branch 'week1' of https://github.com/DecentCat/houseofseasons into week1

mseiser committed:
Merge remote-tracking branch 'upstream/Mainmenu' into week1

mseiser committed:
Merge pull request #5 from DecentCat/mseiser-patch-1

Delete unity/UserSettings directory

MountAnFruh commented:
Move TextMesh Pro Folder to a 3rd party assets folder instead of having it in root ex. 3rdParty/TextMesh Pro
Separate MainMenu stuff into Scripts/Assets folders etc. => Assets/Scripts/MainMenu.cs, Assets/Music/.., Assets/Textures/MainMenu
Make own folder for animations and move player animations in there => Assets/Animations/Player/....

otherwise ok for me

MountAnFruh commented:
also add unity/UserSettings to gitignore

mseiser committed:
Added recommended changes

moved
- player animations to animations folder
- main_menu assets to required folders

added
- 3rd party folder

mseiser committed:
Update .gitignore

mseiser committed:
Update EditorBuildSettings.asset

mseiser commented:
added changes also removed sample scene, changed editor settings in project settings to build again
